docs/interactive_engine/tinkerpop/supported_gremlin_steps.md

```diff
@@ -693,9 +693,9 @@ g.V().out("1..10").with('RESULT_OPT', 'ALL_V')
 g.V().out("1..10").with('RESULT_OPT', 'ALL_V').endV()
 ```
 #### Getting Properites
-The properties of the elements (vertices and/or edges) in the path can be projected by `values()`-step or `valueMap()`-step.
+The properties of the elements (vertices and/or edges) in the path can be projected by `values()`-step, `valueMap()`-step, and `elementMap()`-step.
 It is important to note that the specific elements targeted for property projection are determined by the `RESULT_OPT` setting. 
-For instance, if you configure `RESULT_OPT` as `ALL_V`,  `values()` or `valueMap()` will then project the properties of all vertices present in the path.
+For instance, if you configure `RESULT_OPT` as `ALL_V`,  `values()`, `valueMap()`, or `elementMap()` will then project the properties of all vertices present in the path.
```

Comment:
Add doc for explaining when certain attributes does not exist.

Reply:
Done.